packages/patternlab/styleguide/source/assets/js/vendor.js

```diff
@@ -19,6 +19,7 @@ import 'core-js/fn/array/find-index';
 import 'core-js/fn/array/fill';
 import 'core-js/fn/array/from';
 import 'core-js/fn/array/includes';
+import 'mdn-polyfills/String.prototype.includes'
```

Comment:
At some point we should determine whether these mdn-polyfills imports need semi-colons like the other imports (I think they do). I don't see this causing any functional issues though